packages/react-scripts/config/webpack.config.js

```diff
@@ -701,7 +701,7 @@ module.exports = function(webpackEnv) {
       // NOTE: iModel.js specific plugin to copy a set of static resources from the node_modules
       // directory of each dependent package into the 'build/public' directory.
       // Used for resources such as locales, which are defined by each consuming package.
-      useNewAssetCopy
+      disableNewAssetCopy
```

Comment:
```suggestion
      !disableNewAssetCopy
```

Reply:
I just switched the ordering of the options instead.